packages/blade/package.json

```diff
@@ -127,6 +127,7 @@
     "watch:test:react-native": "yarn test:react-native --watch --onlyChanged",
     "chromatic": "npx chromatic",
     "publish-npm": "node ./scripts/publishToNpm.js",
+    "bundle-size": "node ./scripts/generateBundleSizeInfo.js",
```

Comment:
Done. We don't need to run the entire build, just the `react:prod` build. Added `generate-bundle-size-info` & `pregenerate-bundle-size-info` (this will automatically run before `generate-bundle-size-info`)